examples/question_answering/question_answering/seed_test_run.py

```diff
@@ -0,0 +1,85 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import sys
+from argparse import ArgumentParser
+from argparse import Namespace
+
+import pandas as pd
+from question_answering.evaluator import QuestionAnswerEvaluator
+from question_answering.utils import normalize_string
+from question_answering.workflow import Inference
+from question_answering.workflow import Model
+from question_answering.workflow import TestSample
+from question_answering.workflow import TestSuite
+from question_answering.workflow import ThresholdConfiguration
+
+import kolena
+from kolena.workflow import test
+from kolena.workflow.annotation import ClassificationLabel
+
+MODELS = ["gpt-3.5-turbo-0301", "gpt-3.5-turbo", "gpt-4-0314", "gpt-4"]
+
+
+def main(args: Namespace) -> int:
+    kolena.initialize(os.environ["KOLENA_TOKEN"], verbose=True)
+
+    inference_mapping = {}
+    model_file_path = f"s3://kolena-public-datasets/CoQA/results/{args.model}.csv"
+    df = pd.read_csv(model_file_path)
+
+    # populate inference_mapping
+    for _, row in df.iterrows():
+        data_id = row["data_id"]
+        turn = row["turn"]
+        inference_mapping.setdefault(data_id, {})[turn] = {
+            "answer": row["answer"],
+            "wc_answer": row["wc_answer"],
+            "inference_prompt_tokens": row["inference_prompt_tokens"],
+            "inference_completion_tokens": row["inference_completion_tokens"],
+            "inference_total_tokens": row["inference_total_tokens"],
+        }
+
+    # define a function that generates an inference from a test sample
+    def infer(test_sample: TestSample) -> Inference:
+        result = inference_mapping[test_sample.data_id][test_sample.turn]
+        return Inference(
+            answer=ClassificationLabel(label=result["answer"]),
+            clean_answer=normalize_string(result["answer"]),
+            wc_answer=result["wc_answer"],
+            inference_prompt_tokens=result["inference_prompt_tokens"],
+            inference_completion_tokens=result["inference_completion_tokens"],
+            inference_total_tokens=result["inference_total_tokens"],
+            source=model_file_path,
+        )
+
+    # define configurations
+    configurations = [
+        ThresholdConfiguration(threshold=0.75),
+        ThresholdConfiguration(threshold=0.5),
+    ]
+
+    model = Model(args.model, infer=infer)
+    test_suite = TestSuite.load(args.test_suite)
+    evaluator = QuestionAnswerEvaluator(configurations)
+    test(model, test_suite, evaluator, reset=True)
+    return 0
+
+
+if __name__ == "__main__":
+    ap = ArgumentParser()
+    ap.add_argument("test_suite", help="Name of the test suite to test.")
```

Comment:
Should we have a default value so simply running `poetry run python seed_test_run.py` can give you a test suite / test run?